source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm

```diff
@@ -60,32 +68,40 @@ - (UIView *)renderButtons:(ACRView *)rootView
     ACRRegistration *reg = [ACRRegistration getInstance];
     ACOFeatureRegistration *featureReg = [ACOFeatureRegistration getInstance];
 
-    UIView<ACRIContentHoldingView> *childview = nil;
+    UIStackView *childview = [[UIStackView alloc] init];
+    childview.distribution = UIStackViewDistributionFillProportionally;
     AdaptiveCards::ActionsConfig adaptiveActionConfig = [config getHostConfig] -> GetActions();
-    NSDictionary<NSString *, NSNumber *> *attributes =
-        @{@"spacing" : [NSNumber numberWithInt:adaptiveActionConfig.buttonSpacing],
-          @"distribution" : [NSNumber numberWithInt:UIStackViewDistributionFillProportionally]};
 
     if (ActionsOrientation::Horizontal == adaptiveActionConfig.actionsOrientation) {
-        childview = [[ACRColumnSetView alloc] initWithFrame:CGRectMake(0, 0, superview.frame.size.width, superview.frame.size.height) attributes:attributes];
-        ((ACRColumnSetView *)childview).isActionSet = YES;
```

Comment:
what other parts of the code use the isActionSet property? is it still a good idea to keep this flag existing for something else?

Reply:
This is for layoutsubview method to readjust the action set; this task is handled by ACRContentHoldingUIScrollView atm, but the code can be still used if we others use containing view to the custom view. 

---
In reply to: [387951641](https://github.com/microsoft/AdaptiveCards/pull/3818#discussion_r387951641) [](ancestors = 387951641)